README.md

```diff
@@ -25,12 +25,12 @@ Use this SDK to add real-time video, audio and data features to your Node app. B
 ## Monorepo Navigation
 
 - **Packages**:
-  - [Server SDK](/packages/livekit-server-sdk/README.md)
-  - [Node real-time SDK](/packages/livekit-rtc/README.md)
+  - [Server SDK](https://github.com/livekit/node-sdks/tree/main/packages/livekit-server-sdk) - to interact with server APIs.
```

Comment:
if we change the links to be absolute (I guess for npm?) then we should probably do the same for the examples further down

Reply:
yeah npm was the reason for the change. However, I realized the main README isn't published to npm. So I'm rolling that portion back